docs/man/git-lfs-migrate.1.ronn

```diff
@@ -189,6 +215,18 @@ $ git lfs migrate import --everything --include="*.zip"
 
 Note: This will require a force push to any existing Git remotes.
 
+### Migrate without rewriting local history
+
+You can also migrate files without modifying the existing history of your respoitory:
+
+```
+# Without a specified commit message
```

Comment:
Let's move these comments out of the code block, and replace the shell with 4 spaces, and then `git lfs migrate ...` (and without the `$`).